apps/ui/src/views/Proposal.vue

```diff
@@ -204,146 +204,149 @@ watchEffect(() => {
         </div>
         <router-view :proposal="proposal" />
       </div>
-      <div
+      <Affix
         :class="[
-          'static md:fixed md:top-[72px] md:right-0 w-full md:h-[calc(100vh-72px)] md:max-w-[340px] p-4 md:pb-[88px] border-l-0 md:border-l space-y-4 no-scrollbar overflow-y-scroll',
+          'shrink-0 md:w-[340px] border-l-0 md:border-l',
           { 'hidden md:block': route.name === 'proposal-votes' }
         ]"
+        :top="72"
+        :bottom="88"
```

Comment:
I feel like 88 is too much there. Even 0 is enough I think as our component already has paddings.
This is with 0:
<img width="440" alt="image" src="https://github.com/user-attachments/assets/d8bc13dd-2020-4340-99ac-7652e5a1704a">

This is with 88:
<img width="452" alt="image" src="https://github.com/user-attachments/assets/58fb548e-3e05-4dee-a079-f0ba21ff1604">

@bonustrack

Reply:
Decreased to 64px, which coupled to the component own `p-4`, will result in the original `md:pb-[88px]` that was removed